protocol/chainlib/tendermintRPC.go

```diff
@@ -528,6 +528,11 @@ func (cp *tendermintRpcChainProxy) SendURI(ctx context.Context, nodeMessage *rpc
 		defer res.Body.Close()
 	}
 
+	if res.StatusCode == 504 || res.StatusCode == 429 {
+		utils.LavaFormatError("Received invalid status code", nil, utils.Attribute{Key: "Status Code", Value: res.StatusCode})
+		return nil, "", nil, err
+	}
+
```

Comment:
in URI we need to check json format as well.

Reply:
done : 5fb6ebf93a2aa1aebc13b40e0cfd51fd8d779160